praktikumsplaner-frontend/frontend/src/components/MeldungPraktikumsstelle/MeldungOertlAusbilder.vue

```diff
@@ -0,0 +1,310 @@
+            </v-list>
+            <v-card-actions>
+                <v-spacer />
+                <v-btn
+                    text
+                    @click="cancel()"
+                >
+                    Abbrechen
+                </v-btn>
+                <v-btn
+                    color="primary"
+                    @click="uploadPraktikumsstelle()"
+                >
+                    Hochladen
+                </v-btn>
+            </v-card-actions>
+        </v-card>
+    </v-form>
+</template>
+
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import { useSnackbarStore } from "@/stores/snackbar";
+import { Levels } from "@/api/error";
+import Praktikumsstelle from "@/types/Praktikumsstelle";
+import MeldungService from "@/api/MeldungService";
+import { useRules } from "@/composables/rules";
+import { Dringlichkeit } from "@/types/Dringlichkeit";
+import { YesNo } from "@/types/YesNo";
+import { Ausbildungsjahr } from "@/types/Ausbildungsjahr";
+import { Ausbildungsrichtung } from "@/types/Ausbildungsrichtung";
+import { Studiensemester } from "@/types/Studiensemester";
+import { Studienart } from "@/types/Studienart";
+import { Referat } from "@/types/Referat";
+import router from "@/router";
+
+const praktikumsstelle = ref<Praktikumsstelle>(
+    new Praktikumsstelle("", "", "", "", "")
+);
+const stuzubiSelection = ref<string>();
+const stuzubiSelectionItems = ref<string[]>(["Ausbildung", "Studium"]);
+const isAusbildung = ref<boolean>(false);
+const isStudium = ref<boolean>(false);
+const form = ref<HTMLFormElement>();
+const snackbarStore = useSnackbarStore();
+const validationRules = useRules();
+const requiredRule = [validationRules.notEmptyRule("Darf nicht leer sein!")];
+const emailRule = [
+    validationRules.notEmptyRule("Darf nicht leer sein!"),
+    validationRules.emailRule(),
+];
+
+const ausbildungsRule = computed(() => {
+    return [
+        validationRules.notEmptyRuleAndVisible(
+            isAusbildung.value,
+            "Darf nicht leer sein!"
+        ),
+    ];
+});
+
+const studiumsRule = computed(() => {
+    return [
+        validationRules.notEmptyRuleAndVisible(
+            isStudium.value,
+            "Darf nicht leer sein!"
+        ),
+    ];
+});
+
+function cancel() {
+    form.value?.reset();
+    router.push("/");
+}
+
+function changeSelectedStuzubi() {
+    if (stuzubiSelection.value == stuzubiSelectionItems.value[0]) {
+        isStudium.value = false;
+        isAusbildung.value = true;
+    }
+    if (stuzubiSelection.value == stuzubiSelectionItems.value[1]) {
+        isAusbildung.value = false;
+        isStudium.value = true;
+    }
+}
+
+function uploadPraktikumsstelle() {
+    if (!form.value?.validate()) return;
+    if (isStudium.value) {
+        MeldungService.uploadStudiumsPraktikumsstelle(praktikumsstelle.value)
+            .then(() =>
+                snackbarStore.showMessage({
+                    message: "Praktikumsstelle erfolgreich Angelegt!",
```

Comment:
I think the type of the Praktikumsstelle (Studium or Ausbildung) should be visible inside the message